bench/workgen/workgen.cxx

```diff
@@ -137,6 +137,102 @@ static void *thread_workload(void *arg) {
     return (NULL);
 }
 
+static void *thread_idle_table_cycle_worload(void *arg) {
+
+    WorkloadRunnerConnection *runnerConnection = (WorkloadRunnerConnection *) arg;
+    WorkloadRunner *runner = runnerConnection->runner;
+    WT_CONNECTION *connection = runnerConnection->connection;
+
+    try {
+        runner->start_table_idle_cycle(connection);
+    } catch (WorkgenException &wge) {
+        std::cerr << "Exception while create/drop tables." << std::endl;
+    }
+
+    return (NULL);
+}
+
+int WorkloadRunner::check_timing(const char *name, uint64_t start, uint64_t *stop) {
```

Comment:
See comment later, would it be cleaner to get stop time outside the function and pass it by value. Or maybe even pass the last_interval duration instead.

int WorkloadRunner::check_timing(const char *name, uint64_t last_interval) {

Reply:
Updated.